pyramid_oereb/contrib/data_sources/standard/models/__init__.py

```diff
@@ -174,14 +174,16 @@ class ViewService(base):
             reference_wms (dict of str): The actual url which leads to the desired cartographic
                 representation (multilingual).
             layer_index (int): Index for sorting the layering of the view services for a theme
-            layer_opacity (float): Opacity of a view service
+            layer_opacity (float): Opacity of a view servicA
+            verify_certificate (boolean): Whether the certificate of the WMS shall be verified
         """
         __table_args__ = {'schema': schema_name}
         __tablename__ = 'view_service'
         id = Column(pk_type, primary_key=True, autoincrement=False)
         reference_wms = Column(JSONType, nullable=False)
         layer_index = Column(Integer, nullable=False)
         layer_opacity = Column(Float, nullable=False)
+        verify_certificate = Column(Boolean, nullable=True)
```

Comment:
This would require an effort for all the users that employ the standard models as they would have to change the db structure.

Reply:
Thanks for pointing this out, indeed I think that this should not be required. 
I have adapted the code, and now the standard db model does not require this attribute, we just set it to True by default.